src/views/accounts/AccountChangeSecret/Overview/CardSummary.vue

```diff
@@ -71,6 +72,7 @@ export default {
         {
           title: this.$t('Total'),
           body: {
+            route: { name: 'AccountChangeSecretExecutionList' },
             count: this.data.total_count_change_secret_executions
           }
         },
```

Comment:
The main difference I noticed is that there seems to be an extra comma before "count" in the second object (lines 70 and 78). It should probably look like this:

```
{
   title: this.$t('Total'),
   body: {
      count: this.data.total_count_change_secret_executions
   }
}
```